src/porepy/grids/coarsening.py

```diff
@@ -128,15 +131,21 @@ def tpfa_matrix(g, perm=None, faces=None):
 
     """
     if perm is None:
-        perm = tensor.SecondOrder(g.dim,np.ones(g.num_cells))
+        perm = tensor.SecondOrder(g.dim, np.ones(g.num_cells))
```

Comment:
No, don't think so. Also, in this particular case it doesn't matter, since unit permeability is assigned.